frontend/src/components/ViewTrips/trip.js

```diff
@@ -52,6 +53,8 @@ export function getCollaboratorEmails(collaboratorUidArr) {
  * - tripData: Object holding a Trip document fields and corresponding values.
  * - tripId: Document ID for the current Trip document.
  * - handleEditTrip: Handler that displays the edit trip modal.
+ * - refreshTripsContainer: Handler that refreshes the TripsContainer
+ *        component upon trip creation (Remove when fix Issue #62).
```

Comment:
Is this only upon trip creation? Or is this on trip deletion too?

Reply:
Both, it is called on line 105 as the last line of `deleteTrip` in delete-trip-button.js`.